printer_mods/sloscotty/SW_Wall_Plate/README.md

```diff
@@ -0,0 +1,38 @@
+# Switchwire Wall Plate
+This mod is for pure aesthetics.  A Voron Switchwire logo plate to install on the back wall of your printer's enclosure.
+
+(Or optionally on the front grill if no enclosure - but only if you're not already using the LCD mount...)
+
+![](Images/SW_backwall_plate.jpg)
+
+Note: Overall size is about 225x50 (about 10mm thick)
+
+## BOM
+* (4) M3x8mm BHCS
+* (4) M3xD5.0xH4.0 heat-set inserts
+
+![](Images/printed_pieces.jpg)
+
+## Directions
+1.  Install inserts in the WITCHWIRE_base
+
+![](Images/base_inserts.jpg)
+
+2. Use backplate_and_drillguide to drill holes in your back panel
+
+3. Glue badges into respective recesses on the base (no need to glue base to backplate)
```

Comment:
We don't use glue for any Voron parts or mods.  VHB tape is ok, or screws, or you can make the accent pieces part of the same STL and ask the user to change filament color at a certain layer.